Project_1/Styles.css

```diff
@@ -51,14 +50,21 @@ p {
     line-height: 2.2;
 }
 
-/*i am not sure how to edit either image individually, so right now the Author photo is very much warped /*
+p #pullquote {
+    background-color: #fdcd96;
+    margin-right: -600px;
+    margin-left: -600px;
+    font-size: 25px;
+}
+
+/*i am not sure how to edit either image individually, so right now the Author photo is very much warped */
```

Comment:
You had the right idea with using IDs or classes. If you added an ID or class (I recommend classes) attribute to the figure, like so:
`<figure class="hero-image">` for the octopus photo
and
`<figure class="author-photo">` for Carl's photo, you can modify them independently with CSS.

In the stylesheet, you would reference them like so:
```css
.hero-image {
 /* styles for the octo photo go here */
}
.author-photo {
 /* styles for the octo photo go here */
}
```

Note that the classnames, when you reference them in the CSS, always start with a period, e.g. `.author-photo` -- you don't have to do this in the HTML, where the word class is there to let us know what kind of attribute it is. Classes also cannot have spaces in them, which is why I use the hyphen above.

We'll talk more about this in class this week, but you can also read up on it here: https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Simple_selectors#Class_selectors